packages/patterns/src/types.js

```diff
@@ -22,7 +22,7 @@ export {};
 /** @typedef {import('@endo/marshal').RankCover} RankCover */
 
 /**
- * @typedef {Passable} Key
+ * @typedef {Exclude<Passable, Error | Promise>} Key
```

Comment:
This is still overly broad; we should probably have a TODO.

Reply:
Exclude at the top level doesn't address promises and errors nested further down. Again, I hope we can parameterize Passable.